release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1052,7 +1052,7 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-13-rhv-deprecations"]
 ==== Red Hat Virtualization (RHV) as a host platform for {product-title} will be deprecated
 
-Red Hat Virtualization (RHV) will be deprecated in an upcoming release of {product-title}. Support for {product-title} on RHV will be removed from a future {product-title} release, currently planned as {product-title} 4.14.
+Red Hat Virtualization (RHV) as a host platform for {product-title} is now deprecated, and will be removed in {product-title} 4.14. 
```

Comment:
Should this say future version of OpenShift (and not a version)? 
I know we sometimes say the version and others we don't what are the guidelines for when we say a version and when we don't?

Reply:
Usually we want to avoid naming the version, but since it's a plan for removal, I think it's appropriate in this case.

However, it should still keep the wording about being "planned" as 4.14. I know it's slightly painful, but I'd recommend something like the following:

```suggestion
Red Hat Virtualization (RHV) as a host platform for {product-title} is now deprecated, and is expected to be removed in the next {product-title} release, currently planned as {product-title} 4.14.
```

Or if we are really really sure it's going to be removed, then you could shortcut it to:

```suggestion
Red Hat Virtualization (RHV) as a host platform for {product-title} is now deprecated, and will be removed in the next {product-title} release, currently planned as {product-title} 4.14.
```
Either way, we still need some flexibility in the statement about being planned, because things can change

Reply:
This feature (RHV support) is definitely being removed from the OCP build in 4.14. 
The PMs want this to be clear. Support has ended, the supporting code will be removed in 4.14.

Reply:
In addition, even if someone wished to extend support, there is nobody around anymore to provide it, and there are additional technical reasons that make it difficult or impossible to continue.